web/src/pages/apd/activities/overview/NameAndFundingSource/NameAndFundingSourceForm.js

```diff
@@ -1,8 +1,11 @@
 import { ChoiceList, TextField } from '@cmsgov/design-system';
 import PropTypes from 'prop-types';
-import React, { Fragment, useCallback } from 'react';
+import React, { Fragment, useEffect } from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { joiResolver } from '@hookform/resolvers/joi';
 import { connect } from 'react-redux';
 
+import nameFundingSourceSchema from '../../../../../static/schemas/nameAndFundingSource';
```

Comment:
please put this in the common/schema folder

Reply:
I think Tif wanted it to be referenced by the common module name, so something like-
`import nameAndFundingSourceSchema from '@cms-eapd/common/schemas/nameAndFundingSource';`